src/main/java/com/symphony/ps/pollbot/RealTimeEventComponent.java

```diff
@@ -0,0 +1,45 @@
+package com.symphony.ps.pollbot;
+
+import com.symphony.bdk.core.service.message.MessageService;
+import com.symphony.bdk.core.service.message.exception.PresentationMLParserException;
+import com.symphony.bdk.core.service.message.util.PresentationMLParser;
+import com.symphony.bdk.gen.api.model.*;
+import com.symphony.bdk.spring.events.RealTimeEvent;
+import com.symphony.ps.pollbot.services.PollService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class RealTimeEventComponent {
+
+    @Autowired
+    private PollService pollService;
+
+    @Autowired
+    private MessageService messageService;
+
+    @EventListener
+    public void onMessageSent(RealTimeEvent<V4MessageSent> event) throws PresentationMLParserException {
+        pollService.handleIncomingMessage(
+                event.getSource().getMessage(),
+                StreamType.TypeEnum.fromValue(event.getSource().getMessage().getStream().getStreamType()));
```

Comment:
if the api changes and a new stream type is added this might break
so we could be careful and handle this case and not try to process the event it has an unknown type

Reply:
so I think we should only handle the incoming message from IM and ROOM?